vector-im/element-android
# Makes "Enable Notifications for this session" respond to enabled value in pusher

## Type of change


## Content

Meets criteria:
- the ‘notifications on this device’ toggle should check the pusher.enabled field and display as off when pusher.enabled is false
- When there is a registered pusher with enabled: false - toggling on should set the enabled field to true on the existing pusher

(the other AC are already current behaviour)

## Motivation and context

Epic: https://element-io.atlassian.net/browse/PSG-595
Closes https://element-io.atlassian.net/browse/PSG-785

## Screenshots / GIFs





## Tests

- Disable pusher in device manager
- Go to notification settings
- See Enable Notifications for this session is unchecked
- Check this setting
- See pusher is now enabled

## Tested devices

- OS version(s): Android 13

## Checklist
